src/reducers/index.js

```diff
@@ -10,4 +10,19 @@ const photosApp = combineReducers({
   mangoIndexByDate
 })
 
+export const mustShowSelectionBar = state => state.ui.showSelectionBar || state.ui.selected.length !== 0
+
+export const getPhotosByMonth = ({ photos }) => {
+  let months = {}
+  photos.map(p => {
+    const month = p.created_at.slice(0, 7) + '-01T00:00'
```

Comment:
I know it's not perfect, but I couldn't find a better way :/ I added some comments..